src/test/mock/mockRawls.js

```diff
@@ -0,0 +1,69 @@
+export const mockRawls = {
+  workspacesList(success, _) {
```

Comment:
I think this response would need to be an option rather than the only value. I may want to use a few versions of this response depending on what I'm testing.

Reply:
We can make multiple mock implementations